paper.rst

```diff
@@ -205,8 +205,9 @@ devices, such as graphics processing units (GPUs), tensor processing units (TPUs
 and field-programmable gate arrays (FPGAs).
 
 **JIT compiler support.** Standardized APIs and behavior should be amenable to
-just-in-time (JIT) compilation and graph-based optimization (e.g., PyTorch (TODO: citation),
-JAX (TODO: citation), and TensorFlow (TODO: citation)). For APIs and behavior
+just-in-time (JIT) and ahead-of-time (AOT) compilation and graph-based
```

Comment:
mentioning AOT here because it applies as well and the AOT abbreviation is used further on in the paper.